docs/install.rst

```diff
@@ -9,8 +9,7 @@ It will get you to a point of having a local running instance.
 .. warning::
 
     Read the Docs does not itself run under Python 3 (though it does support
-    building documentation for Python 3 projects). Please ensure the subsequent
-    steps are performed using Python 2.7.
+    building documentation for Python 3 projects).
```

Comment:
I suppose that we need to remove the whole warning and update the docs to point the contributors to use Python 3.6